app/src/main/java/protect/card_locker/LoyaltyCardViewActivity.java

```diff
@@ -350,12 +366,16 @@ public void onStopTrackingTouch(SeekBar seekBar) {
         });
 
         binding.mainImage.setOnClickListener(view -> onMainImageTap());
+
         // This long-press was originally only intended for when Talkback was used but sadly limiting
         // this doesn't seem to work well
         binding.mainImage.setOnLongClickListener(view -> {
             setMainImage(true, true);
             return true;
         });
+
+        binding.mainImageDescription.setOnLongClickListener(view -> copyBarcodeToClipBoard());
```

Comment:
I think this would be better to move directly below `binding.mainImageDescription.setOnClickListener` so everything of `binding.mainImageDescription` is in one place.